tests/Unit/DataStructures/Test_OrientVariables.cpp

```diff
@@ -208,6 +209,159 @@ void test_3d_orient_variables() noexcept {
   CHECK(number_of_orientations_checked == 48);
 }
 
+// Test 0D slice of a 1D element
```

Comment:
I have the same general concern/request that testing the coords and a scalar would make the test a lot better if the code needs optimizations later :)